src/pages/api/v1/get-api-key.ts

```diff
@@ -0,0 +1,63 @@
+import { type NextApiRequest, type NextApiResponse } from "next";
+import CryptoJS from "crypto-js";
+
+import { PROFILES } from "../../../utils/constants";
+import { apiSupabaseClient } from "../../../utils/supabaseServerClient";
+
+export default async function handler(
+	request: NextApiRequest,
+	response: NextApiResponse,
+) {
+	if (request.method !== "GET") {
+		response.status(405).json({ error: "Method not allowed" });
+		return;
+	}
+
+	const supabase = apiSupabaseClient(request, response);
+
+	const {
+		data: { user },
+		error: userError,
+	} = await supabase.auth.getUser();
+
+	if (userError || !user) {
+		response.status(401).json({ error: "Unauthorized user" });
+		return;
+	}
```

Comment:
@rogerantony-dev Add console.warn here - https://timeless.slack.com/archives/C09139Z0Y75/p1763630308244649

Follow the rules from this @docs/api_logging_rules.md. See other migrated endpoints on how it was done.